src/nvim/option.c

```diff
@@ -2440,11 +2440,9 @@ did_set_string_option (
     if (briopt_check(curwin) == FAIL)
       errmsg = e_invarg;
   }
```

Comment:
The comment would then be in a block to which it does not relate directly. I feel like this is wrong on some level.